package.json

```diff
@@ -216,59 +239,59 @@
         },
         {
           "command": "magit.branching",
-          "when": "editorLangId == magit"
+          "when": "activeEditor"
```

Comment:
I'm assuming `activeEditor` is needed for magit to figure out the repository it should use (based on #56), so I kept that.

Reply:
Yeah, the commandPrimer uses a simpler/faster uri based approach to resolve the current repo, compared to magit.status which finds the repo no matter what :D